_includes/functions/sort-rename.md

```diff
@@ -0,0 +1,17 @@
+{% capture /dev/null %}
+<!-- rename things for sorting purposes -->
+{% assign _result = include.name %}
+{% if include.name contains "BIP" %}
```

Comment:
I don't think you need to special case this:

```diff
diff --git a/_includes/functions/sort-rename.md b/_includes/functions/sort-rename.md
index 5499852..b573545 100644
--- a/_includes/functions/sort-rename.md
+++ b/_includes/functions/sort-rename.md
@@ -2,16 +2,12 @@
 <!-- rename things for sorting purposes -->
 {% assign _result = include.name %}
 {% if include.name contains "BIP" %}
-  {% if include.name == "BIP70 payment protocol" %}
-    {% assign _result = include.name | replace: "BIP70", "BIP0070" %}
-  {% else %}
-    {% assign _strlen = include.name | size %}
-    {% case _strlen %}
-      {% when 4 %}{% assign _result = include.name | replace: "BIP", "BIP000" %}
-      {% when 5 %}{% assign _result = include.name | replace: "BIP", "BIP00" %}
-      {% when 6 %}{% assign _result = include.name | replace: "BIP", "BIP0" %}
-      {% else %}{% include ERROR675_BIP_STRING_DOESNT_MATCH_EXPECTED_PATTERN %}
-    {% endcase %}
-  {% endif %}
+  {% assign _strlen = include.name | split: ' ' | first | size %}
+  {% case _strlen %}
+    {% when 4 %}{% assign _result = include.name | replace: "BIP", "BIP000" %}
+    {% when 5 %}{% assign _result = include.name | replace: "BIP", "BIP00" %}
+    {% when 6 %}{% assign _result = include.name | replace: "BIP", "BIP0" %}
+    {% else %}{% include ERROR675_BIP_STRING_DOESNT_MATCH_EXPECTED_PATTERN %}
+  {% endcase %}
 {% endif %}
 {% endcapture %}{{_result}}
```

Reply:
@jnewbery noice!  Thanks!  Will push an edit.